client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -88,6 +90,74 @@ def _get_artifact_msg(self, key):
     def _get_artifact(self, key):
         raise NotImplementedError
 
+    def set_schema(self, input: dict, output: dict = None) -> None:
+        """
+        Sets the input and output schemas of this Deployable Entity. Schemas are stored as
+        model artifacts.
+
+        The output schema is optional.
+
+        To validate a prediction's input and output against these schemas, use the
+        :func:`~verta.registry.validate_schema` decorator.
+
+        Parameters
+        ----------
+        input : dict
+            Input schema as an OpenAPI-compatible json dict. Easiest to create using pydantic.BaseModel.schema() [#]_.
+        output : dict
+            Output schema as an OpenAPI-compatible json dict. Easiest to create using pydantic.BaseModel.schema().
+
+
+        References
+        ----------
+        .. [#] https://docs.pydantic.dev/1.10/usage/schema/
+
+        """
+        if not isinstance(input, dict):
+            raise TypeError(
+                f"`input` must be of type dict, not {type(input)}; did you remember to call `.schema()`?"
+            )
+        if output is not None and not isinstance(output, dict):
+            raise TypeError(
+                f"`output` must be of type dict, not {type(output)}; did you remember to call `.schema()`?"
+            )
+
+        schema = {
+            "input": input,
+        }
+        if output is not None:
+            schema["output"] = output
+
+        # write a temp file because otherwise `log_artifact` will think the artifact contents are the file path
+        with tempfile.NamedTemporaryFile(mode="w") as temp_file:
+            temp_filename = temp_file.name
+            with open(temp_filename, "w") as file:
+                json.dump(schema, file)
+            self.log_artifact(
+                key="model_schema_json", artifact=temp_filename, overwrite=True
+            )
+
+    def get_schema(self) -> Dict[str, dict]:
+        """
+        Gets the input and output json schemas of this Deployable Entity, in the format:
+
+        .. code-block:: python
+        {
+            "input": <input schema>,
+            "output": <output schema>
+        }
```

Comment:
```suggestion
        .. code-block:: python

            {
                "input": <input schema>,
                "output": <output schema>
            }
```
This needs a blank line and slight change to indentation to render correctly.  Otherwise it looks like 
![image](https://github.com/VertaAI/modeldb/assets/114943931/e7b13de1-3703-4b97-80a5-21b307c94c7b)